src/server/api/data.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+
+export const getCoordinates = async (postcode: any): Promise<Coordinates> => {
+  const response = await axios.get(
```

Comment:
Can you bit a bit more precise about what we expect this HTTP request to return? The axios library uses generic methods `get<T>` so that `response.data` will be of type T. I would define a `PostcodeResponse` type and do `get<PostcodeResponse>` to be more accurate.

Reply:
updated code - https://github.com/builtbybea/bass-and-treble/pull/2/commits/7f5ddd2e7fc027b1d555901210b718fc7ff00a9a